index.rst

```diff
@@ -40,6 +40,7 @@ See what's new in the :doc:`Release Notes <releases/v11_0/notes>`.
    :caption: Developer Guide
    :name: part-developer-guide
 
+   development/decision_process.rst
```

Comment:
And, as for DM-4658:
* It's not clear to me that this belongs under "Science Pipelines". This document applies to the entire DM team.
* There are no indications in the document itself that it is an official policy statement or subject to change control. Admittedly the original Confluence document and RFC-53 didn't have such either, but at least it was in a "Standards and Policies" area of the hierarchy. Including this in an lsst-sqre repo that also contains general documents means that there is no easy way to impose such change control using GitHub permissions.